### DISCUSSION TITLE
History kann nicht aufgerufen werden.

### DISCUSSION THREAD

**Discussion by Kitegerd** [RaBa64/OpenDTU#9] [General]
Version   dd3c12d

Wenn man auf die Datenpunkt der einzelnen Tage klickt, ändert sich das Datendiagramm darüber nicht. 
Funktioniert es bei euch?

**Comment by RaBa64** [comment]
Die beiden Diagramme sind voneinander unabhängig.
Oben wird also immer nur der letzte Tag angezeigt.
Aber das wäre natürlich noch eine interessante Erweiterung. Die Daten sind ja da.
Als nächstes versuche ich die Google Charts durch FOSS zu ersetzen.
